apps/docs/app/globals.css

```diff
@@ -12,12 +12,12 @@
 
 html,
 body {
-  font-family: var(--default-font-family);
-  font-size: var(--default-font-size);
+  font-family: var(--hd-default-font-family);
+  font-size: 16px;
```

Comment:
doing so stops users from changing their font size in their browsers.

Reply:
Indeed, however, it is there to define a base relative to the design, the accessibility features are always available through the zoom in the browser.